docs/zh_cn/multi_modal/internvl.md

```diff
@@ -1,3 +1,114 @@
-# InternVL
 
-TODO
+# internvl
+
+## 简介
+
+InternVL2 是由OpenGVLab开源的一个强大的开源视觉语言模型（VLM）. LMDeploy 已在PyTorch后端支持 InternVL2 系列模型 [OpenGVLab/InternVL2](https://huggingface.co/collections/OpenGVLab/internvl-20-667d3961ab5eb12c7ed1463e) , 这里以 InternVL2-26B 模型[OpenGVLab/InternVL2-26B](https://huggingface.co/OpenGVLab/InternVL2-26B)为例来说明如何使用通lmdeploy框架部署InternVL系列模型；
+
+### 环境配置及镜像构建
+
+安装docker以及docker-compose以及[compose-gpu-support](https://docs.docker.com/compose/gpu-support/),并确保您的宿主机cuda版本，若宿主机的cuda大版本为cuda11，请确保其高于11.8；若宿主机的cuda大版本为cuda12，请确保其高于12.2；若镜像内的cuda版本高于宿主机的cuda版本，镜像内的Pytorch将无法使用；完成环境配置后，在lmdeploy项目根目录执行：
+
+```shell
+#build the base image,it only use to base LLM
+docker build -t openmmlab/lmdeploy:latest-cu12 . -f ./docker/Dockerfile
```

Comment:
why build it again

Reply:
> why build it again

for one person who use lmdeploy fisrt, i suggest them building image by theirself bacause it can avoid some bug about pip package version;  first build base image, and second build internVL image base on base image